tests/test_integrations.py

```diff
@@ -0,0 +1,40 @@
+import os
+from dotenv import load_dotenv
+from langfuse.integrations import openai
+from langfuse.api.client import FintoLangfuse
+from langfuse.version import __version__ as version
+
+from tests.utils import create_uuid
+
+
+load_dotenv()
+
+api = FintoLangfuse(
+    environment=os.environ["HOST"],
+    username=os.environ["LF_PK"],
+    password=os.environ["LF_SK"],
+)
+
+
+def test_openai_chat_completion():
+    trace_id = create_uuid()
+    completion = openai.ChatCompletion.create(
```

Comment:
I just tested this out locally. openai is shown on my end as type any in the IDE, whereas when using the original openai SDK, i get autocompletions. Is there a way to solve that?

Reply:
I didn't understand exactly, what's the issue @maxdeichmann

Reply:
![image](https://github.com/langfuse/langfuse-python/assets/17686849/34411c9f-57bc-4d92-8f50-cc702bad7608)

create is of type `any` here. If you import `import openai` into `integrations.py`, this should be typed. Could you add that?

Reply:
Here @maxdeichmann for me its same, so how do I fix it
![image](https://github.com/langfuse/langfuse-python/assets/57898986/41d38976-f0af-4dc5-8916-bc5406ecc74e)
![image](https://github.com/langfuse/langfuse-python/assets/57898986/4e50a29e-39ae-474c-9bba-a162277ef66f)